lib/screens/assets/images.dart

```diff
@@ -0,0 +1,12 @@
+const imgSvgFireman = "assets/fireman.svg";
+const imgSvgIconAlarm = "assets/ico_alarm.svg";
+const imgSvgIconFire = "assets/ico_fire.svg";
+const imgSvgIconWatch = "assets/ico_watch.svg";
```

Comment:
can you place all the icons under an `icons/` folder?

Reply:
I'll create a new file under `assets` called `icons.dart`.

Reply:
But a quick question, do you want the `const` statements in a separate file or the actual svg files in a different folder at the root?
So the root would have
```
- assets
|___ icons
```

Reply:
sorry i meant the actual files

Reply:
Added the correct folder for assets in the latest commit